src/intelligence_layer/examples/qa/multiple_chunk_qa.py

```diff
@@ -29,11 +29,13 @@ class MultipleChunkQaInput(BaseModel):
             Can be arbitrarily long list of chunks.
         question: The question that will be answered based on the chunks.
         language: The desired language of the answer. ISO 619 str with language e.g. en, fr, etc.
+        explainability_enabled: Whether to generate highlights (using the explainability feature) for the answer;  default False for performance reasons
```

Comment:
I think i prefer an actively phrased name better, such as `generate_highlights`. Also calling it highlights is more descriptive

Reply:
Indeed, it makes more sense, as it directly influences how the highlights are being generated. Fixed it here -> https://github.com/Aleph-Alpha/intelligence-layer-sdk/commit/9aefe00ba8b901afe5b1d179a2ec2b7e60cdf391